src/run/run_step.go

```diff
@@ -20,6 +21,37 @@ var log = logging.MustGetLogger("run")
 
 // Run implements the running part of 'plz run'.
 func Run(graph *core.BuildGraph, label core.BuildLabel, args []string) {
+	run(graph, label, args, false)
+}
+
+// Parallel runs a series of targets in parallel.
+func Parallel(graph *core.BuildGraph, labels []core.BuildLabel, args []string) {
+	var g errgroup.Group
+	for _, label := range labels {
+		label := label // capture locally
+		g.Go(func() error {
+			return run(graph, label, args, true).Wait()
+		})
+	}
+	if err := g.Wait(); err != nil {
+		log.Fatalf("Command failed: %s", err)
```

Comment:
nope, we have lost it at this point. I see `plz run parallel` as pretty fire-and-forget though...